src/components/common/CodeReview.js

```diff
@@ -218,29 +250,47 @@ class CodeReview extends Component {
     return comment.type !== COMMENT;
   }
 
-  renderCommentList(commentList) {
+  renderCommentChild(comments, parentId) {
+    const { classes, isFeedbackView } = this.props;
     const { focusedId } = this.state;
-    return commentList.map((comment) => (
-      <tr key={comment._id} className="comment">
-        <td className="comment editor" colSpan={2}>
-          <CommentEditor
-            comment={comment}
-            readOnly={this.getReadOnlyProperty(comment)}
-            focused={focusedId === comment._id}
-            onEditComment={(_id) => this.handleEdit(_id)}
-            onDeleteComment={(_id) => this.handleDelete(_id)}
-            onCancel={this.handleCancel}
-            onSubmit={(_id, line, content) =>
-              this.handleSubmit(_id, line, content)
-            }
-          />
-        </td>
-      </tr>
+    const levelComments = comments
+      .filter((comment) => comment.data.parent === parentId)
+      .sort((comment) => comment.created_at);
```

Comment:
I'm pretty sure it's `createdAt`. You can check in the network tab.

Reply:
Yes you are right !